src/Dialog.js

```diff
@@ -93,8 +104,19 @@ Dialog.propTypes = {
   children: PropTypes.node.isRequired,
   isOpen: PropTypes.bool.isRequired,
   onDismiss: PropTypes.func.isRequired,
-  theme: PropTypes.object,
-  title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired
+  theme: PropTypes.object
+}
+
+DialogHeader.defaultProps = {
+  alignItems: 'center',
+  bg: 'gray.1',
+  justifyContent: 'space-between',
```

Comment:
Why are we defaulting to `space-between`?

Reply:
Ah this was left over, probably makes sense to remove this now since the close button is no longer in the header